src/infra/factory/usecases/add-manufacturer-factory.ts

```diff
@@ -0,0 +1,9 @@
+import { AddManufacturerUseCase } from "../../../application/usecases/manufacturer";
+import { LoggerService } from "../../logging/logger";
+import { ManufacturerRepository } from "../../repositories/manufacturer";
+
+export const makeDbAddManufacturer = (): AddManufacturerUseCase => {
```

Comment:
```suggestion
export const makeDbAddManufacturer = (): AddManufacturerUseCase => {
```

Aqui você está gerando acoplamento entre o caso de uso e a implementação, o ideal seria isso ser uma interface

Reply:
Você poderia tbm criar uma pasta só para essas factories, da controller e também dos use cases

Reply:
[exemplo](https://github.com/rmanguinho/clean-ts-api/blob/db12990a7b6e4b478074ade9cd89b2e539ad3931/src/main/factories)